wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1296,6 +1297,56 @@ def workflow_status_with_date(workflow_state):
     return _("%(status_display)s %(task_name)s %(started_at)s") % translation_context
 
 
+@register.inclusion_tag(
+    "wagtailadmin/shared/keyboard_shortcuts_dialog.html", takes_context=True
+)
+def keyboard_shortcuts_dialog(context):
+    user_agent = context["request"].headers.get("User-Agent", "")
+    is_mac = re.search(r"Mac|iPod|iPhone|iPad", user_agent)
+    modifier = "⌘" if is_mac else "Ctrl"
+    shortcuts = {
+        ("common-actions", _("Common Actions")): [
+            {"label": _("Copy"), "shortcut": f"{modifier} + c"},
+            {"label": _("Cut"), "shortcut": f"{modifier} + x"},
+            {"label": _("Paste"), "shortcut": f"{modifier} + v"},
+            {
+                "label": _("Paste without formatting"),
```

Comment:
The mac wording for this label is "Paste and Match Style".

Not sure how close we want to stay to OS style icons/wording.

Reply:
How about `Paste text only`?

Reply:
I think Windows-speak is "Paste without formatting" and on Mac it is "Paste and Match Style".
The feature seems the same: Whatever snippet is in the clipboard, drop the styling, but match the current style in the text editor.

I assume (don't have Windows here), that if you use "Paste without formatting" into a heading element, the text still gets the heading elements style applied???

My suggestion is, to align shortcuts and labels (wording) with the OS.

Something like:

``` python

mac = "mac"
win = "windows"
os = mac if re.search(r"Mac|iPod|iPhone|iPad", user_agent) else win

shortcuts = {
    ("common-actions", _("Common Actions")): [
        ...,
        {
            "label": _("Paste and Match Style") if os == mac else _("Paste without formatting"),
            "shortcut": f"{modifier} + shift + v",
        },
        # You'd need something similar for the other shortcut.
        {
            "label": _("Redo"),
            "shortcut": f"Shift + {modifier} + z"  if os == mac else f"{modifier} + y",
        },
        ...,
    ]
}
```

Reply:
This ^ can become quite ugly code. So we can also do something like this:
``` python
# Windows is the default.
shortcuts = [
    # Win/Mac are the same.
    {
        "label": "Some label",
        "shortcut":  "Some shortcut",
    },
    # Mac has other label
    {
        "label": "Some label",
        "label_mac": "Some Mac label",
        "shortcut":  "Some shortcut",
    },
    # Mac has other shortcut
    {
        "label": "Some label",
        "shortcut": "Some shortcut",
        "shortcut_mac": "Some Mac shortcut",
    },
    # Mac has other label and shortcut
    {
        "label": "Some label",
        "label_mac": "Some Mac label",        
        "shortcut": "Some shortcut",
        "shortcut_mac": "Some Mac shortcut",
    },
]

# Update the labels and shortcuts with the Mac variant if we are on OSX or iOS.
if re.search(r"Mac|iPod|iPhone|iPad", user_agent):
    for item in shortcuts:
        item["label"] = item.pop("label_mac", item["label"])
        item["shortcut"] = item.pop("shortcut_mac", item["shortcut"])


print(shortcuts)
```

Reply:
@lb- @allcaps I understand the required changes, but do we have any list or thing for these changes? For example, what should the label name be on Mac and Windows? Right now, I only know that "Paste and Match Style" and "Paste without formatting" need to be changed, but I don't know about other shortcuts.

Reply:
I have made changes to this specific "Paste and Match Style". Are there any more labels that need to be changed based on OS?

Reply:
Also, there should be some changes in unit tests to check this change. Should I update the unit tests in this PR itself? Or can we ignore it for now?

Reply:
@lb- : D